src/demos/Sliders.vue

```diff
@@ -186,5 +214,9 @@ import Accordion from "./../components/Accordion.vue";
 export default class Sliders extends Vue {
   value = 50;
   data = [1, 50, 100];
+
+  updateValue(value) {
+    console.log(value);
```

Comment:
leftover console log here

Reply:
This is in the documentation file only, but thanks for noticing